misc/config_definitions.json

```diff
@@ -4429,6 +4429,13 @@
         "xirrus_disable_stations": {
             "default": false,
             "type": "boolean"
+        },
+        "permission.device_group.user_allow_dynamic": {
```

Comment:
This file should be alphabetical otherwise it will be a mess.

Reply:
It got out of order after a rebase, thanks for fixing